kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -0,0 +1,74 @@
+### Domain lifecycle sample scripts
+
+The operator provides sample scripts to start up or shut down a specific Managed Server or cluster in a deployed domain, or the entire deployed domain.
+
+**Note**: Prior to running these scripts, you must have previously created and deployed the domain.
+
+These scripts can be helpful when scripting the life cycle of a WebLogic Server domain. For information on how to start, stop, restart, and scale WebLogic Server instances in your domain, see [Domain Life Cycle](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle).
+
+#### Scripts to start and stop a Managed Server
+The `startServer.sh` script starts a Managed Server by patching the `spec.managedServers.<server-name>.serverStartPolicy` attribute of the domain resource to `ALWAYS`. For clustered servers, it also increases the `spec.clusters.<cluster-name>.replicas` value for the Managed Server's cluster by `1`. The script provides an option to keep the `spec.clusters.<cluster-name>.replicas` value constant for clustered servers. See the script `usage` information by using the `-h` option.
+
+```
+$ startServer.sh -d domain1 -n weblogic-domain-1 -s managed-server1
+[INFO] Patching start policy of server 'managed-server1' from 'NEVER' to 'ALWAYS' and incrementing replica count for cluster 'cluster-1'.
+domain.weblogic.oracle/domain1 patched
+[INFO] Successfully patched server 'managed-server1' with 'ALWAYS' start policy!
+
+       The replica count for cluster 'cluster-1' updated to 1.
+```
+
+The `stopServer.sh` script shuts down a running Managed Server by patching the  `spec.managedServers.<server-name>.serverStartPolicy` attribute of the domain resource to `NEVER`. For clustered servers, it also decreases the `spec.clusters.<cluster-name>.replicas` for the Managed Server's cluster by `1`. The script provides an option to keep the `spec.clusters.<cluster-name>.replicas` value constant for clustered servers. See the script `usage` information by using the `-h` option.
```

Comment:
Small point... These JSONPath samples are not correct because the managedServers and clusters elements are lists rather than maps. You might use something like `spec.managedServers[?(serverName="<server-name>")].serverStartPolicy` or `spc.clusters[?(clusterName="<cluster-name>")]`

Reply:
ok. I'm wondering if this might be difficult to read for someone who is not familiar with various JSON data structures. Can we can specify something like `spec.managedServers[<server-name>].serverStartPolicy` to keep it slightly short instead of specifying exact syntax? Thanks.

Reply:
Sure, that's fine.

Reply:
Thanks. I have made the changes. Please let me know if I missed something.